migrations/01-create-artist-table.sql

```diff
@@ -0,0 +1,6 @@
+ -- migrations/01-create-artist-table.sql
+  CREATE TABLE Artists (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(255) NOT NULL,
+  genre VARCHAR(255) NOT NULL
+  );
```

Comment:
Things like the names of the files should not be commented at the top of the files, it just adds noise so its always worth going through your work before submitting it to make sure things like this dont creep through by accident. Also the indentation here is a little bit off:
```
  CREATE TABLE Artists (
    id SERIAL PRIMARY KEY,
    name VARCHAR(255) NOT NULL,
    genre VARCHAR(255) NOT NULL
);
```